src/services/serviceRequest.js

```diff
@@ -1,8 +1,9 @@
-import request from 'request-promise-native';
+import request from 'axios';
 
 export async function serviceRequest(requestInfo) {
   try {
-    const response = request(requestInfo);
+    // eslint-disable-next-line no-unused-vars
```

Comment:
How come eslint complains on this, I don't see any no unused vars

Reply:
(_) => true